server/game/AbilityContext.ts

```diff
@@ -17,6 +17,7 @@ interface AbilityContextProperties {
     events?: any[];
     stage?: Stages;
     targetAbility?: any;
+    duel?: any;
```

Comment:
I don't think we should need duel on the context object... Why is this necessary?